### PR TITLE
[MB-4107] Storybook components for OrdersTable and ProfileTable

### PR DESCRIPTION
**Pull request by Ronolibert**
## Description

Add storybook components for OrdersTable and ProfileTable for use in review page later

## Setup
`make storybook`

## Code Review Verification Steps
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-4107) for this change

## Screenshots

image
image

### CODE REVIEW

**Review by mariaclaratgolfo — CHANGES_REQUESTED** [review]
This is looking great. Would you be open to pairing for 15m to design QA together? Can't inspect on my local but want to:

- Be sure we're using the 3 column grid for mobile
- Be sure it's working on mobile and desktop
- inspect padding and text styles

The only thing that appears off without inspecting is the lines are breaking between the columns. I wonder what the difference is between [this implementation ](https://storybook.dp3.us/?path=/story/components-stacked-table--default)and yours that allows the dividing lines to stay connected?

**Review by mariaclaratgolfo — APPROVED** [review]

File: src/components/Customer/Review/Review.module.scss [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  border-collapse: separate;
+  border-radius: 4px;
+  border: 1px solid $base-lighter;
+  border-spacing: 0;
+  width: 100%;
+  @include u-padding-left('05');
+  @include u-padding-right('05');
+
+  th {
+    font-weight: bold;
```
Comment (Shkeating) [inline src/components/Customer/Review/Review.module.scss:57]: based on the screenshots in the JIRA ticket, it looks like these should not be black. They use #3d4551, which is `$base-darker`. so just add that in here and I think you're good to go!

`color: $base-darker;`

Comment (Shkeating) [reply src/components/Customer/Review/Review.module.scss:57]: I checked in the spec and was able to verify this in [the design](https://app.abstract.com/projects/1f6c1fd0-88a2-11e9-aaf7-a7ce5896598b/branches/master/commits/aad3abc8090de43ebf9db91e4ad2d21a844b1293/files/4fa2c8bc-9dd1-4d09-b713-0bdc28483118/layers/976FA82B-79BA-4333-80D2-D73556E0D7F6?collectionId=ac5fb495-6a26-4019-a7e2-8b7df985bfd6&collectionLayerId=1f808449-035d-4ce1-932a-976245f0e013&mode=build&selected=3112447098-0AE490E9-B5CC-489D-BF45-DCFB57924CB6):

image

**Review by Shkeating — COMMENTED** [review]
lookin good, just need to adjust some small things :)

File: src/components/Customer/Review/Review.module.scss (hunk 1 above)
Comment (Ronolibert) [reply src/components/Customer/Review/Review.module.scss:57]: Ahh, didn't realize. Will change this!

Comment (Ronolibert) [reply src/components/Customer/Review/Review.module.scss:57]: Changed this to the correct color

**Review by Shkeating — APPROVED** [review]

**Pull request merged**